lib/env_bang.rb

```diff
@@ -24,8 +24,12 @@ def use(var, *args)
     end
 
     def raise_formatted_error(var, description)
+      # We do not want errors while running rake tasks
+      return if $PROGRAM_NAME =~ /rake$/
```

Comment:
I worried about all other rake tasks. @yard  your thoughts ?

Reply:
In the case of a rake task `$PROGRAM_NAME` will be eq to `/usr/local/rvm/rubies/ruby-3.0.2/bin/rake`
nothing more task-specific was found